tests/AppBundle/Command/ProcessEventCommandTest.php

```diff
@@ -270,4 +271,39 @@ private function jobFinishedSpec()
             ]);
         static::assertEquals(0, count($jobs));
     }
+
+    /**
+     * Creates a new job, this time with EventCategory's on the Event.
```

Comment:
no apostrophe needed

Reply:
PHPStorm complains `EventCategorys` is a spelling error, but I'll just add it to the dictionary :) I didn't want to put `EventCategories` because that isn't the class name.

Reply:
Yeah I'm never sure what to do in that situation either. I think though that the `ies` is probably correct, but it's clear either way. Just not the possessive! :)